src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -1052,6 +1052,13 @@ export function submitAppointmentOrRequest(history) {
           type: FORM_SUBMIT_SUCCEEDED,
         });
 
+        recordEvent({
+          event: `${GA_PREFIX}-vaos-community-care-submission`,
```

Comment:
This event looks sort of like a duplicate of the event on line 1015. I think we'd want to add this new data to that event and/or the submission-successful even below (similar to how we use the `additionalData` object).

Reply:
Done.